DataGenerator/DataGenerator.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, mastersheet, label, config, keys, train_transform = None, val
 
         # Convert regression value to histogram class
         regression_y = mastersheet[label].to_numpy()
-        bins = np.arange(np.min(regression_y), np.max(regression_y)-2, 3)
```

Comment:
Please remove this, I don't see the point of transforming regression value to an histogram.